httpd-proxy/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN chmod +x /usr/local/bin/httpd-foreground
 
 EXPOSE 80 443
 
-CMD ["httpd-foreground"]
\ No newline at end of file
+CMD ["httpd-foreground.sh"]
```

Comment:
Don't the COPY directives, etc, have to be updated?

Reply:
oops, good catch.  That was a last-minute change to make it possible for Karen (Windows) to build without having to manually change files from `CRLF` to `LF`